### PR TITLE
Leaderboard Entry count and Dates, Score added to Submissions

### PR DESCRIPTION
**Pull request by lotusgraham**
image

image

### CODE REVIEW

**Comment by lotusgraham** [comment]
image

**Comment by ckcollab** [comment]
I merged in team stuff and this branch still has a ton of weird changes??

File: codalab/apps/web/models.py [hunk 1]
```diff
@@ -900,7 +900,7 @@ def sortkey(x):
             for header in headers:
                 header['subs'].sort(key=sortkey, reverse=False)
             # compute total column span
-            column_span = 2
+            column_span = 4
```
Comment (ckcollab) [inline codalab/apps/web/models.py]: Good eye finding this!

File: codalab/apps/teams/templates/teams/edit.html [hunk 2]
```diff
@@ -46,7 +47,7 @@
     <h3><b>Team Information</b></h3>
     <div class="row">
         <div class="col-sm-12">
-            <table class="table table-striped affiliation">
+            <table id="teams-edit" class="table table-striped affiliation">
```
Comment (ckcollab) [inline codalab/apps/teams/templates/teams/edit.html]: Hmm, maybe you just made this change on this branch? I think that's fine, I merged in develop for you so as long as teams looks/behaves fine, I'm OK with this being in the PR :)

File: codalab/apps/web/templates/web/competitions/_results_page.html [hunk 3]
```diff
@@ -88,6 +89,8 @@
                                     {{scoredata.username}}
                                 {% endif %}
                             </td>
+                            <td> {{scoredata.count}} </td>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_results_page.html]: annoying nitpick incoming: will you pull the spaces around this and put spaces inside the braces so it looks like... `<td>{{ scoredata.count }}</td>`

**Comment by lotusgraham** [comment]
I'm also seeing some weird stuff from the teams branch here, which is weird since I'm 100% sure I branched directly from develop. I'm not sure how that gets in there. What's the best way to get rid of it?

**Comment by ckcollab** [comment]
NOTE: This is waiting on some fixes for the score column in new submission problem..

**Comment by ckcollab** [comment]
Awesome! Please fix the conflicts and I'll test this sucker out

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 4]
```diff
@@ -85,6 +85,7 @@
             <thead>
                 <tr>
                     <th>#</th>
+                    <th>Score</th>
                     <th>Filename</th>
                     <th>Score</th>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: There are actually 2 columns?

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 5]
```diff
@@ -119,6 +121,7 @@
                             <input type="hidden" name="state" value='0' />
                         {% endif %}
                         <td>{{ forloop.counter }}</td>
+                        <td>{{submission_info.score}}</td>
                         <td>{{submission_info.filename}}</td>
                         <td>{{ submission_info.score }}</td>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: Yeah another column here?

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 6]
```diff
@@ -119,6 +121,7 @@
                             <input type="hidden" name="state" value='0' />
                         {% endif %}
                         <td>{{ forloop.counter }}</td>
+                        <td>{{submission_info.score}}</td>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: spaces after `{{ `

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html (hunk 4 above)
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: I still see 2 columns here?

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 7]
```diff
@@ -119,10 +121,11 @@
                             <input type="hidden" name="state" value='0' />
                         {% endif %}
                         <td>{{ forloop.counter }}</td>
-                        <td>{{submission_info.filename}}</td>
                         <td>{{ submission_info.score }}</td>
-                        <td>{{submission_info.submitted_at|date:"m/d/Y H:i:s"}}</td>
-                        <td class="statusName">{{submission_info.status_name}}</td>
+                        <td>{{ submission_info.filename }}</td>
+                        <td>{{ submission_info.score }}</td>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: Duplicated!

**Pull request merged**